src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3060,7 +3066,7 @@ void addProperties(ImmutableArray<ParameterSymbol> recordParameters)
 #endif
             }
 
-            static bool hidesInheritedMember(Symbol symbol, NamedTypeSymbol type)
+            static Symbol? getInheritedMember(Symbol symbol, NamedTypeSymbol type)
```

Comment:
>getInheritedMember [](start = 27, length = 18)

What's the benefit of returning a `Symbol` rather than a `bool`? It looks like we're only using the value to test for `null`.

Reply:
Hmm, I was using the `Symbol` at one point, but not now. Reverted.